.github/workflows/test_openvino_examples.yml

```diff
@@ -22,9 +22,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.10"]
+        python-version: ["3.8", "3.11"]
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
why are we moving to Ubuntu 22? is it aligned with OpenVINO OS support matrix?

Reply:
24.04 is around the corner and 20.04 is pretty old now.  22.04 is officially supported and we release archives for it too. NPU is not supported with 20.04, so 22.04 is arguably the best version at the moment.

But if there is any reason to keep 20.04 happy to move it back.

Reply:
Thanks for the context. It makes sense.